### PR TITLE
Multitables v3

### PR DESCRIPTION
**Pull request by anton-matosov**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Awesome!

File: src/modules/systemlib/mixer/multi_tables [hunk 1]
```diff
@@ -1,135 +1,197 @@
-#!/usr/bin/tclsh
```
Comment (thomasgubler) [inline src/modules/systemlib/mixer/multi_tables]: why has this file no file ending?

Comment (anton-matosov) [inline src/modules/systemlib/mixer/multi_tables]: What do you mean by encoding?

Comment (thomasgubler) [inline src/modules/systemlib/mixer/multi_tables]: I mean file-*ending*. See https://github.com/PX4/Firmware/pull/1594

**Pull request merged**